src/pudl/metadata/resources/sec10k.py

```diff
@@ -254,34 +287,61 @@
         "etl_group": "sec10k",
         "field_namespace": "sec",
     },
-    "core_sec10k__parents_and_subsidiaries": {
+    "out_sec10k__parents_and_subsidiaries": {
```

Comment:
`out_sec10k__parents_and_subsidiaries` now flows from the `core_sec10k__quarterly_exhibit_21_company_ownership` table, not the `raw_sec10k__parents_and_subsidiaries` table which is created upstream. We only use that raw parents and subs table to get the results of record linkage between filers and EIA utilities. It felt like we should get rid of this `core` table and just have the `core_sec10k__quarterly_exhibit_21_company_ownership` and subsequent `out_sec10k__parents_and_subsidiaries` tables